packages/babel-helpers/src/helpers.ts

```diff
@@ -788,7 +788,13 @@ helpers.iterableToArrayLimit = helper("7.0.0-beta.0")`
     var _d = false;
     var _s, _e;
     try {
-      for (_i = _i.call(arr); !(_n = (_s = _i.next()).done); _n = true) {
+      _i = _i.call(arr);
+      if (i === 0) {
+        if (typeof _i !== "object" && typeof _i !== "function") return;
```

Comment:
Use `Object(_i) !== _i`, which checks for both functions and objects but it excludes null.

Reply:
Fixed!